Project2015To2017Tests/ProjectWriterTest.cs

```diff
@@ -142,11 +143,11 @@ public void SkipDelaySignNull()
 			var writer = new ProjectWriter();
 			var xmlNode = writer.CreateXml(new Project
 			{
-				DelaySign = null,
+				AdditionalPropertyGroups = new[] {new XElement("PropertyGroup")},
 				FilePath = new FileInfo("test.cs")
 			});
 
-			var delaySign = xmlNode.Element("PropertyGroup").Element("DelaySign");
+			var delaySign = xmlNode.Elements("PropertyGroup").Skip(1).First().Element("DelaySign");
```

Comment:
The first `PropertyGroup` is the one created by `ProjectWriter`. `DelaySign` is from `AdditionalPropertyGroups`, so we need to skip the first node.